slack/welcome-message.txt

```diff
@@ -43,11 +43,14 @@ You can invite your friends by sharing this link with them: https://keep-network
 7. Have fun and ask questions if you have them!
 
 *For more information and other social channels:*
+
+* _Whitepaper (EN)_ - https://keep.network/whitepaper
+* _Whitepaper draft (RU)_ - https://backend.keep.network/whitepaper-ru
+* _Random Beacon yellowpaper PDF_ - http://docs.keep.network/random-beacon/index.pdf
```

Comment:
Any reason why we wouldn't just link to the HTML version at `http://docs.keep.network/random-beacon/`?

Reply:
In my mind pdf is more shareable, but if you think you'd rather do HTML, I have no problem with that :)

Reply:
My only thought is HTML doesn't require downloading anything… Meh, I dunno, let's just leave it as is for now.